### PR TITLE
New reservation

### PR DESCRIPTION
**Pull request by byiringiroscar**
#### What does this PR do ❓ 
This PR is raised to implement the following features 🆗 :

- [x] Add a new reservation page
### How should it be tested ⬇️ ⬇️ ⬇️ 

clone this repository
     
```
git clone https://github.com/shasherazi/laptops-ecommerce
```
Navigate to folder
```
cd laptops-ecommerce
```
Switch to this new-reservation
```
git switch new-reservation
```
Run project and Installation
```
npm install
```
Start project
```
npm run dev
```

### Thanks for reviewing my PR 😄

### CODE REVIEW

**Review by shasherazi — APPROVED** [review]
Everything looks good, ready to merge :shipit:

File: src/pages/Reservation.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const Reservation = () => {
+    return (
+        <div className='reserve-section'>
+            <h1>Book a vespa Testa-Ride</h1>
+            <hr/>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Necessitatibus, ad.</p>
+            <div className="button-container">
+                <Link  to="#!" className="">London</Link >
+                <Link  to="#!" className="">Reserve</Link >
+            </div>
```
Comment (imranhossainemi) [inline src/pages/Reservation.jsx:14]: This part is needed to be a form field with select option of different city or town name , quantity input field with default value 1 and validate the input with isnan only positive.  Reserve is a submit button.

File: src/pages/Reservation.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            <img className='search' src={search} />
```
Comment (imranhossainemi) [inline src/pages/Reservation.jsx:15]: Remove this feature we do not need this.

File: src/pages/ReservationPage.jsx [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
 import { fetchReservations, removeReservation } from '../redux/reservations/reservationSlice';
 import './ReservationPage.css';
```
[the remark this answers is not in the document]
Comment (imranhossainemi) [reply src/pages/ReservationPage.jsx:5]: @shasherazi I have fixed typo in my new PR

File: src/style/home.css [hunk 4]
```diff
@@ -6,6 +6,12 @@
 #97BF0F
 
 */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
```
Comment (imranhossainemi) [inline src/style/home.css:14]: @byiringiroscar Please remove this code I am not using it . It make some design part bad

File: src/style/home.css [hunk 5]
```diff
@@ -123,8 +128,9 @@ header nav ul li:hover .single-nav {
...
 
   .main-content {
-    width: 80%;
+    width: calc(100vw - 150px);
     position: absolute;
-    left: 15%;
+    left: 150px;
+    padding-top: 0;
   }
 }
```
Comment (imranhossainemi) [inline src/style/home.css:136]: @byiringiroscar I change this previous values . You need to change values again with git conflicts

**Review by imranhossainemi — CHANGES_REQUESTED** [review]
## Code Review for  Signup/login page #26 

### General Comments:

- [ ] The code follows the project's coding guidelines and standards.
- [ ] The changes are clear and easy to understand.
- [ ] The code is adequately commented for better readability and understanding.
- [ ] The overall logic and implementation approach seem appropriate is not right in some places.

### Major Observations:

### Specific Feedback:

#### 1. Please checkout comments on design changes  for this project. 

### Positive Feedback:
React components and routes are working perfectly.  

### Closing Notes:
Please make sure you are using web content accessibility guidelines.

File: src/pages/ReservationPage.jsx [hunk 6]
```diff
@@ -25,7 +26,9 @@ export default function ReservationPage() {
     <section className='reservation'>
       <h1>Reservation List</h1>
       <div className='reservation-new'>
-        <button type='button'> Abb New Reservation</button>
+      <Link to="/new-reservation">
+            <button type='button'>Add New Reservation</button>
+          </Link>
```
Comment (imranhossainemi) [inline src/pages/ReservationPage.jsx:31]: This link button do not work

**Pull request merged**